src/com/TTT/Game.java

```diff
@@ -3,12 +3,14 @@
 public class Game {
 
     private Board board;
+    private TurnUI turns;
     private int currentPlayer;
     private String player1Marker;
     private String player2Marker;
 
-    public Game(Board board) {
+    public Game(Board board, TurnUI turns) {
         this.board = board;
+        this.turns = turns;
         currentPlayer = 1;
         player1Marker = PlayerMarkers.X.symbol();
```

Comment:
For info, you could statically import `PlayerMarkers` so that you have `player1Marker = X.symbol()`.